metricbeat/scripts/mage/template/metricsetDoc.tmpl

```diff
@@ -11,6 +11,7 @@ This file is generated! See scripts/mage/docs_collector.go
 {{end -}}
 include::../../../{{.Mod.Path | basePath}}/{{.Mod.Base}}/{{.Metricset.Title}}/_meta/docs.asciidoc[]
 
+{{if .Metricset.IsDefault}}This is a default metricset. If the host module is unconfigured, this metricset is enabled by default.{{end}}
```

Comment:
Nit. It should be possible to avoid the empty lines added when metricset is not default with something like this:
```suggestion
{{if .Metricset.IsDefault -}}
This is a default metricset. If the host module is unconfigured, this metricset is enabled by default.
{{end -}}
```

Reply:
Uh, those newlines get me every time.

Reply:
So, all this seems to do is remove the newline when the metricset _is_ default.

Reply:
Nevermind, it somehow removes _two_ newlines as opposed to one.